libs/Microsoft.MixedReality.WebRTC/ISignalingService.cs

```diff
@@ -0,0 +1,221 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Text;
+
+namespace Microsoft.MixedReality.WebRTC
+{
+    public enum SignalingType
```

Comment:
Note: we should add that enum to the API instead of carrying around strings which are error-prone.

Reply:
Logged as #188, can be done independently - it's an API break so it should be timed carefully.